docs/features/caching.rst

```diff
@@ -42,5 +42,5 @@ If you want to add your own caching method implement the following interfaces an
 
 ``IOcelotCache<FileConfiguration>`` this is for caching the file configuration if you are calling something remote to get your config such as Consul.
 
+Thanks Mohsen Rajabi, PR `1167 <https://github.com/ThreeMammals/Ocelot/pull/1167>`_ add IMemoryCache Asp Core.

```

Comment:
**First,**
The sentence contains English grammar mistakes!

**Second,**
Tom mentions people efforts by PR or Issue links in docs, but he doesn't mention real contributor's name!

**Third,**
It should be pasted after line 46 actually.

**Finally,**
A lack of information to introduce in docs. More wider and clear description of the related feature should be written.